packages/shared/src/modules/projects/libs/enums/projects-api-path.enum.ts

```diff
@@ -1,4 +1,5 @@
 const ProjectsApiPath = {
+	BY_ID: "/:id",
```

Comment:
We used '$ID' instead of 'BY_ID before'. I suppose it's better to keep it unified